tests/models/roberta_prelayernorm/test_modeling_roberta_prelayernorm.py

```diff
@@ -0,0 +1,547 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+
+from transformers import RobertaPreLayerNormConfig, is_torch_available
+from transformers.testing_utils import TestCasePlus, require_torch, slow, torch_device
+
+from ...generation.test_utils import GenerationTesterMixin
+from ...test_configuration_common import ConfigTester
+from ...test_modeling_common import ModelTesterMixin, floats_tensor, ids_tensor, random_attention_mask
+
+
+if is_torch_available():
+    import torch
+
+    from transformers import (
+        RobertaPreLayerNormForCausalLM,
+        RobertaPreLayerNormForMaskedLM,
+        RobertaPreLayerNormForMultipleChoice,
+        RobertaPreLayerNormForQuestionAnswering,
+        RobertaPreLayerNormForSequenceClassification,
+        RobertaPreLayerNormForTokenClassification,
+        RobertaPreLayerNormModel,
+    )
+    from transformers.models.roberta_prelayernorm.modeling_roberta_prelayernorm import (
+        ROBERTA_PRELAYERNORM_PRETRAINED_MODEL_ARCHIVE_LIST,
+        RobertaPreLayerNormEmbeddings,
+        create_position_ids_from_input_ids,
+    )
+
+
+class RobertaPreLayerNormModelTester:
```

Comment:
I know we might not be doing this for a lot of model, but I think that if we can have copied from wherever we just copy pasted previous tests, would help to know what reviewers actually need to pay a lot of attention to! Might be a nit tho. WDYT @sgugger